flytekit/taskplugins/sqlite/task.py

```diff
@@ -0,0 +1,119 @@
+import contextlib
+import os
+import sqlite3
+import tempfile
+import typing
+from dataclasses import dataclass
+from enum import Enum
+
+from flytekit import FlyteContext, TaskMetadata, kwtypes
+from flytekit.annotated.base_sql_task import SQLTask
+from flytekit.plugins import pandas as pd
+from flytekit.types import FlyteFile, FlyteSchema
+from flytekit.types.flyte_file import unarchive_file
+
+
+@dataclass
+class SQLite3Config(object):
+    """
+    Use this configuration to configure if sqlite3 files are passed in dynamically into the task or
+    to configure a path to a static sqlite3 compatible database file that is available either
+      - within the container
+      - or from a publicly downloadable source
+
+    If the Mode is marked dynamic, then a variable called ``sqlite: FlyteFile[".sqlite"]`` will be automatically added
```

Comment:
I dunno, I never really like it when we did this in Presto. I feel like magically adding inputs is weird.  Can we maybe make this two different tasks?

Reply:
probably you are right, ya adding a new task should be cheap, will do